kafka-ui-react-app/src/redux/reducers/topics/__test__/reducer.spec.ts

```diff
@@ -1,4 +1,8 @@
-import { MessageSchemaSourceEnum, TopicColumnsToSort } from 'generated-sources';
+import {
+  MessageSchemaSourceEnum,
+  SortOrder,
```

Comment:
Why do we have Enum suffix in some cases but do not in others? @workshur

Reply:
🤷‍♂️